plugins/animate/animate.cpp

```diff
@@ -128,6 +134,11 @@ struct animation_hook : public animation_hook_base
             view->unref();
         }
     }
+
+    animation_hook(const animation_hook &) = default;
```

Comment:
Copy is deleted anyway because of unique_ptr, let's mark it like that. Same applies to all other places in this PR.